src/app/shared/components/fy-userlist/fy-userlist-modal/fy-userlist-modal.component.ts

```
@@ -70,26 +58,26 @@ export class FyUserlistModalComponent implements OnInit, AfterViewInit {
       return acc;
     }, {});
   }
 
-  getSeparatorKeysCodes() {
+  getSeparatorKeysCodes(): number[] {
     return [ENTER, COMMA];
   }
 
-  addChip(event: MatChipInputEvent) {
+  addChip(event: MatChipInputEvent): void {
     if (event && event.chipInput) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)